apps/portal/src/app/features/file.service.ts

```diff
@@ -16,12 +16,12 @@ export class FileService {
     private graphqlService: GraphqlService,
     private apollo: Apollo,
   ) {
-    this.jobIdsSubject.next(JSON.parse(localStorage.getItem('jobIds') || '[]'));
+    this.jobIdsSubject.next(JSON.parse(sessionStorage.getItem('jobIds') || '[]'));
```

Comment:
When are job ids removed in this case?

Reply:
as soon as user closed the tab Job Ids are removed from storage